src/test/java/hudson/plugins/build_timeout/impl/ElasticTimeOutStrategyTest.java

```diff
@@ -20,6 +25,29 @@
  */
 public class ElasticTimeOutStrategyTest extends TestCase {
 
+    private File tempDir;
+
+    private synchronized File getTempDir() {
+        if (tempDir == null) {
+            tempDir = Files.createTempDir();
+        }
+        return tempDir;
+    }
+
+    private synchronized void deleteTempDir() throws IOException {
+        if (tempDir != null) {
+            FileUtils.deleteDirectory(tempDir);
+            tempDir = null;
+        }
+    }
+
+
+    @Override
+    protected void tearDown() throws Exception {
+        super.tearDown();
+        deleteTempDir();
```

Comment:
`deleteOnExit` didn't work as expected.
I decided to delete the directory in `tearDown`.